evasquez26/angular4-startup
# S3 One 36 37 Course class creation and edition

## Story / Task

[ONE-36](https://bixlabs.atlassian.net/browse/ONE-36) and [ONE-37](https://bixlabs.atlassian.net/browse/ONE-37)

## What this PR does?

Allows the user to create and edit classes for a course.

### Before 

- We could not create or edit a class for a course

### After

- We can create or edit a class for a course

### How to test / reproduce

- Login into the app
- Go to create course section
- Create a course
- Observe that there's a new classes tab
- Add or edit different classes for the course
- They should be ordered from the nearest to start to the latest

### TODOs

N/A

### Related PRs

N/A

## Migrations

NO
